ui/app/templates/components/calendar-widget.hbs

```diff
@@ -59,46 +57,72 @@
     {{#if this.showCalendar}}
       <div class="calendar-widget-container" data-test-calendar-widget-container>
         <div class="select-year">
-          <Hds::Button
-            data-test-previous-year
-            id="previous-year"
-            disabled={{this.disablePastYear}}
-            {{on "click" this.subYear}}
-            @color="secondary"
-            @text="previous year"
-            @icon="chevron-left"
-            @isIconOnly={{true}}
-          />
-          <p data-test-display-year>
+          {{#if this.disablePastYear}}
+            <Hds::TooltipButton
+              @text={{concat
+                (sub this.displayYear 1)
+                " is unavailable because it is before your start date. Change your start month to a date in "
+                (sub this.displayYear 1)
+                " to see data for this year."
+              }}
+              aria-label="More information about why previous year is disabled"
+            >
+              <Hds::Button
+                data-test-previous-year
+                id="previous-year"
+                disabled={{true}}
+                {{on "click" this.subYear}}
```

Comment:
nit: I think we can remove the click event for a disabled button - not necessary, just minor cleanup.